py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -1,10 +1,182 @@
+from typing import Callable, Dict, Set
 import torch
-from torch._decomp import register_decomposition, core_aten_decompositions
+from torch._decomp import (
+    register_decomposition,
+    core_aten_decompositions,
+    get_decompositions as get_torch_decompositions,
+)
 
+aten = torch.ops.aten
 
-DECOMPOSITIONS = {**core_aten_decompositions()}
+_core_aten_decompositions: Dict[
+    torch._ops.OpOverload, Callable
+] = core_aten_decompositions()
+enabled_decompositions: Set[torch._ops.OpOverload] = {
```

Comment:
If a contributor is writing a converter for one of the `enabled_decompositions`, they should remove it from the set.

Reply:
Is there any sort of checking to capture this?

Reply:
Currently, there would only be a runtime check for this. Specifically, the `get_decompositions` function invokes a check that there are no duplicate registrations across the different decomposition registries.

Reply:
We might want to think about on registration checking